make/conf/module-loader-map.conf

```diff
@@ -91,3 +91,50 @@ PLATFORM_MODULES= \
 PLATFORM_MODULES_windows= \
     jdk.crypto.mscapi \
     #
+
+NATIVE_ACCESS_MODULES= \
```

Comment:
Hello Jan, does this make configuration allow for something like:

```
NATIVE_ACCESS_MODULES= ${BOOT_MODULES} \
${PLATFORM_MODULES} \
foo.bar.additional.module
```
(please ignore any syntax issues in that snippet)

to make the intention clear as well as reduce the chances of missing some boot or platform module in this NATIVE_ACCESS_MODULES?

Reply:
> to make the intention clear as well as reduce the chances of missing some boot or platform module in this NATIVE_ACCESS_MODULES?

The list to be be granted native access is a subset, it shouldn't be granted all modules mapped to the boot or platform class loaders.

Reply:
I see. I then misunderstood a part of the PR description.

Reply:
Hello Jan, my suggestion was based on a misunderstanding that NATIVE_ACCESS_MODULES is always a superset of boot and platform modules. What you currently have looks fine to me.

Reply:
And also, for the record, the .conf files are supposed to be in a syntactical subset compatible with both bash and make syntax, so this would not have been possible anyway. :)